lib/pushNotifications.js

```diff
@@ -18,14 +17,12 @@ export async function registerForPushNotificationsAsync() {
   }
 
   // Stop here if the user did not grant permissions
-  if (finalStatus !== 'granted') {
-    return;
-  }
+  if (finalStatus !== 'granted') return;
 
   // Get the token that uniquely identifies this device
   let token = await Notifications.getExpoPushTokenAsync();
 
-  console.log('user token', token);
+  cb(token);
```

Comment:
No need for callback, you can return the token and `await registerForPushNotificationsAsync()` when consuming this function.